services/blockchain-indexer/Dockerfile

```diff
@@ -17,7 +17,9 @@ RUN npm ci
 
 FROM node:18-alpine
 
-RUN adduser -D lisk && \
+RUN apk update && apk add --no-cache mysql-client && \
```

Comment:
Could you please pin the version for `mysql-client`? Please use fuzzy versioning only with major version number.